src/applications/personalization/profile/tests/e2e/personal-health-care-contacts/contacts.cypress.spec.js

```diff
@@ -21,8 +19,7 @@ describe('Personal health care contacts -- feature enabled', () => {
     ];
     mockGETEndpoints(otherEndpoints, 200, {});
 
-    featureToggles = generateFeatureToggles({ profileContacts: true });
-    cy.intercept('GET', '/v0/feature_toggles*', featureToggles);
+    cy.intercept('GET', '/v0/feature_toggles*', generateFeatureToggles());
```

Comment:
Remove "feature enabled" from test title
`describe('Personal health care contacts -- feature enabled',`

Reply:
I hope the authexp team will forgive me for this sin, as it feels silly to ask for a re-review for this text deletion.